sky/packages/sky/lib/src/gestures/tap.dart

```diff
@@ -11,7 +11,9 @@ import 'events.dart';
 import 'pointer_router.dart';
 import 'recognizer.dart';
 
-typedef void GestureTapCallback();
+typedef void GestureTapCallback(ui.Point globalPosition);
+typedef void GestureTapDownCallback();
+typedef void GestureTapCancelCallback();
```

Comment:
Why not just pass an event in each of these callbacks? Then we wouldn't have to add parameters in the future if someone cared about, say, the event timestamp.